sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -238,7 +239,7 @@ def create_experiment(
 
 
 @pytest.fixture
-def katib_client():
+def katib_client_create_experiment():
```

Comment:
Do we need separate clients for each tests ?
E.g. for Training, we use single TrainingClient(): https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client_test.py#L254

Reply:
Oh, maybe we do not need to separate clients for these two tests now.